examples/nginx-centos7/s2i/bin/save-artifacts

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+#
+# S2I save-artifacts script for the 'nginx-centos7' image.
+# The save-artifacts script streams a tar archive to standard output.
+# The archive contains the files and folders you want to re-use in the next build.
+#
+# For more information see the documentation:
+#	https://github.com/openshift/source-to-image/blob/master/docs/builder_image.md
+#
+tar cf - /usr/share/nginx/html/
```

Comment:
it's correct functionally, it's not correct logically because you'd never save/restore your html files in this way.  you save/restore content you pulled down in your assemble script.  the html files are coming out of the git repo (presumably).